no-comma-dangle rule reports wrong line number

eslint 0.4.4 on

```
var a = {
  method: function() {

  },
};
```

reports

```
  2:2   error  Trailing comma                  no-comma-dangle
```

While I would expect it to report 4th line for having a trailing comma.

--- a/lib/rules/no-comma-dangle.js
+++ b/lib/rules/no-comma-dangle.js
@@ -24,7 +24,7 @@ module.exports = function(context) {
         // The last token in an object/array literal will always be a closing
         // curly, so we check the second to last token for a comma.
         if(secondToLastToken.value === "," && items.length && lastItem) {
-            context.report(lastItem, "Trailing comma.");
+            context.report(lastItem, secondToLastToken.loc.start, "Trailing comma.");
         }
     }
 

--- a/tests/lib/rules/no-comma-dangle.js
+++ b/tests/lib/rules/no-comma-dangle.js
@@ -24,6 +24,7 @@ eslintTester.addRuleTest("lib/rules/no-comma-dangle", {
     invalid: [
         { code: "var foo = { bar: \"baz\", }", errors: [{ message: "Trailing comma.", type: "Property"}] },
         { code: "foo({ bar: \"baz\", qux: \"quux\", });", errors: [{ message: "Trailing comma.", type: "Property"}] },
-        { code: "var foo = [ \"baz\", ]", errors: [{ message: "Trailing comma.", type: "Literal"}] }
+        { code: "var foo = [ \"baz\", ]", errors: [{ message: "Trailing comma.", type: "Literal"}] },
+        { code: "var foo = { bar: \"bar\"\n\n, }", errors: [{ message: "Trailing comma.", line: 3}] }
     ]
 });

